torch/csrc/api/include/torch/linalg.h

```diff
@@ -36,20 +36,20 @@ inline Tensor& eigvalsh_out(Tensor& result, const Tensor& self, std::string uplo
   return torch::linalg_eigvalsh_out(result, self, uplo);
 }
 
-inline Tensor norm(const Tensor& self, optional<Scalar> opt_ord, optional<IntArrayRef> opt_dim, bool keepdim, optional<ScalarType> opt_dtype) {
-  return torch::linalg_norm(self, opt_ord, opt_dim, keepdim, opt_dtype);
+inline Tensor norm(const Tensor& self, optional<Scalar> opt_ord, optional<IntArrayRef> opt_dim, bool keepdim, optional<ScalarType> opt_dtype, bool flatten) {
+  return torch::linalg_norm(self, opt_ord, opt_dim, keepdim, opt_dtype, flatten);
 }
 
-inline Tensor norm(const Tensor& self, std::string ord, optional<IntArrayRef> opt_dim, bool keepdim, optional<ScalarType> opt_dtype) {
+inline Tensor norm(const Tensor& self, std::string ord, optional<IntArrayRef> opt_dim, bool keepdim, optional<ScalarType> opt_dtype, bool flatten) {
```

Comment:
Why would the norm entries be edited? Also this particular entry ignores the flatten argument it's given.